src/components/Tag/Tag.css

```diff
@@ -0,0 +1,84 @@
+.defaultTag{
+    background-color: var(--sun-glow);
+
+    height: 50;
+    width: 140;
+ 
+    border-radius: 20px;
+    border: none;
+    
+    
+    text-align: center;
+    color: var(--baby-powder);
+    font-family: 'DM Sans';
+    font-size: 30px;
+    font-weight: bold;
+}
+
+.cuisineTag{
+    background-color: var(--sun-glow);
+
+    height: 50;
+    width: 140;
+ 
+    border-radius: 20px;
+    border: none;
+    
+    
+    text-align: center;
+    color: var(--baby-powder);
+    font-family: 'DM Sans';
+    font-size: 30px;
+    font-weight: bold;
+}
+
+.priceTag{
+    background-color: var(--orange);
+
+    height: 50;
+    width: 140;
+ 
+    border-radius: 20px;
+    border: none;
+    
+    
+    text-align: center;
+    color: var(--baby-powder);
+    font-family: 'DM Sans';
+    font-size: 30px;
+    font-weight: bold;
+}
+
+.allergenTag{
+    background-color: var(--indian-red);
+
+    height: 50;
+    width: 140;
+ 
+    border-radius: 20px;
+    border: none;
+    
+    
+    text-align: center;
+    color: var(--baby-powder);
+    font-family: 'DM Sans';
+    font-size: 30px;
+    font-weight: bold;
+}
+
+.mealTag{
+    background-color: var(--fuchsia-rose);
+
+    height: 50;
+    width: 140;
+ 
+    border-radius: 20px;
+    border: none;
+    
+    
+    text-align: center;
+    color: var(--baby-powder);
+    font-family: 'DM Sans';
+    font-size: 30px;
+    font-weight: bold;
+}
```

Comment:
This is a lot of repetition. To avoid this, we can assign a tag to multiple classes. Consider making a className=`tag {tagType}Tag` in javascript. Then in CSS, we can do:
.tag {
   height: 50;
    width: 140;

    border-radius: 20px;
    border: none;


    text-align: center;
    color: var(--baby-powder);
    font-family: 'DM Sans';
    font-size: 30px;
    font-weight: bold;
}
and for each tagtype, we can just change the background-color. for example, 
.mealTag {
   background-color: var(--fuchsia-rose);
}